recurseFibonacci.c

```diff
@@ -0,0 +1,33 @@
+//Clarke Littlejohn
+//recursive fibonacci
+#include<stdio.h>
+void fib(int[],int,int);
+
+int main(){
+	
+	int fTerm;
```

Comment:
Mostly right, but your code doesn't handle edge cases of when the user asks for only 0 or 1 fibonacci numbers - a seg fault occurs because of the fibAry[0] = 1; and fibAry[1] = 1; lines.